README.md

```diff
@@ -1,107 +1,168 @@
-## Real Dev Squad Mobile App
+# Real Dev Squad Mobile App
 
 Welcome to the repository of our mobile app, which mirrors all the features of our web applications.
-## Tech Stack
 
-   * React Native
-   * TypeScript
-   * StyleSheet
-## About the Project
+## 📱 Tech Stack
 
-This project serves as the Real Dev Squad cross-platform mobile app, built using React Native.
-## First Phase
+- React Native
+- TypeScript
+- StyleSheet
 
-- Github Authentication ✅ (we did it via webview, requirement has changed to browser login from github)
-- QR code Scanner Authentication ✅
-- Three Screens - Home and Profile ✅
-- Home screen Features ✅
-- Feature with switching from I am idle to doing task ✅
-- Feature with switching from out of office from back again ✅
-- Profile Screen in which we can change our profile through our phone’s gallery ✅
-- API integration in Profile Screen ✅
+## 📖 About the Project
 
-## Second Phase
-- Three Screens - Home , Goals and Profile ✅
-- Home screen Features ✅
-- Goals site Tab
-- Goals site feature will have the TODO app where you can assign the task to the member of the RDS.
+This project serves as the Real Dev Squad cross-platform mobile app, built using React Native. It aims to provide a seamless experience for our users by replicating the features of our web application.
 
-Feel free to download our RDS app by clicking on ➡️ [RDS App](https://drive.google.com/file/d/1aEv941izi5bOJzttZj22EKl47jTnsYy_/view?usp=sharing)
+## 🚀 Features
 
-**Note** : Please delete the previous RDS app build if you have.
+### First Phase
 
+- **Github Authentication** ✅ (originally done via WebView, now moved to browser login for GitHub)
+- **QR Code Scanner Authentication** ✅
+- **Three Screens:** Home and Profile ✅
+- **Home Screen Features** ✅
+- **Status Switch:** Switch between “I am idle” to “Doing task” ✅
+- **Availability Toggle:** Switch between “Out of Office” and “Back Again” ✅
+- **Profile Screen:** Modify profile details using your phone’s gallery ✅
+- **API Integration in Profile Screen** ✅
 
-## Prerequisites
+### Second Phase
 
+- **Three Screens:** Home, Goals, and Profile ✅
+- **Home Screen Features** ✅
+- **Goals Tab:** Displays the TODO app where you can assign tasks to the members of the Real Dev Squad.
 
```

Comment:
We can remove this from the readme file. Could you take other's opinions as well?

Reply:
@ashifkhn what are your thoughts on this?